### PR TITLE
Update ObjectNav code path

### PR DESCRIPTION
**Pull request by srama2512**
## Motivation and Context

The current ObjectNav code does not run on Stretch. This PR updates the ObjectNav code path with the latest changes in other parts of the library. 

## Changelog

* Reduce dependence on `import open3d` working
* fix import bug for `PybulletIKSolver`
* comment out joint positions in stretch_object_nav_env.py`
* fix sophus vs. xyt bugs in two places

## How Has This Been Tested

Tested locally on Stretch RE1

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/home_robot/home_robot/utils/image.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def to_o3d_point_cloud(xyz, rgb=None, mask=None):
 
 def show_point_cloud(xyz, rgb=None, orig=None, R=None):
     # http://www.open3d.org/docs/0.9.0/tutorial/Basic/working_with_numpy.html
+    import open3d as o3d
```
Comment (cpaxton) [inline src/home_robot/home_robot/utils/image.py:162]: can you add an issue (tag me) to remove this entirely? it should be done in my new PR https://github.com/facebookresearch/home-robot/pull/58

we also have a duplicate point_cloud.py in utils

File: src/home_robot/home_robot/utils/image.py [hunk 2]
```diff
@@ -143,6 +142,7 @@ def depth_to_xyz(depth, camera: Camera):
 
 def to_o3d_point_cloud(xyz, rgb=None, mask=None):
     """conversion to point cloud from image"""
+    import open3d as o3d
```
Comment (cpaxton) [inline src/home_robot/home_robot/utils/image.py:145]: this is another duplicate function - can be filed as an issue

**Review by cpaxton — APPROVED** [review]
LGTM! Would be great to have your detailed install/troubleshooting notes in a separate PR

I noted a couple places where it would be useful to file an issue (and tag me!)

**Comment by cpaxton** [comment]
Confirming this works on my robot as well.
We should remove the "input()" statements that block action execution
@exhaustin we need to add an option for a blocking wait when executing move actions

File: src/home_robot/home_robot/utils/image.py (hunk 2 above)
Comment (srama2512) [reply src/home_robot/home_robot/utils/image.py:145]: Created issue here: https://github.com/facebookresearch/home-robot/issues/65

File: src/home_robot/home_robot/utils/image.py (hunk 1 above)
Comment (srama2512) [reply src/home_robot/home_robot/utils/image.py:162]: @cpaxton - `import open3d as o3d` can fail on some older CPUs. It's worth making this import optional. This is not handled in https://github.com/facebookresearch/home-robot/pull/58 yet.

Comment (cpaxton) [reply src/home_robot/home_robot/utils/image.py:163]: interesting - why is that?
i think we should only import open3d in point cloud specific code (like point_cloud.py)

**Pull request merged**